pkg/otlp/config.go

```diff
@@ -104,6 +104,11 @@ func IsEnabled(cfg config.Config) bool {
 	return hasSection(cfg, config.OTLPReceiverSubSectionKey)
 }
 
+// HasLogsSection checks if OTLP logs are explicitly enabled in a given config.
```

Comment:
What if users have `logs: enabled: false`? Does HasLogsSection also return true? If so that isn't what we expected.

Reply:
Good catch, fixed